cascade/src/main/java/me/saket/cascade/CascadePopupMenu.kt

```diff
@@ -52,7 +57,8 @@ open class CascadePopupMenu @JvmOverloads constructor(
     val background: () -> Drawable? = { null },
     val menuList: (RecyclerView) -> Unit = {},
     val menuTitle: (MenuHeaderViewHolder) -> Unit = {},
-    val menuItem: (MenuItemViewHolder) -> Unit = {}
+    val menuItem: (MenuItemViewHolder) -> Unit = {},
+    val overlayColor: () -> Int? = { null }
```

Comment:
> Should there be a flag to enable this particular feature or just check the nullability of resource value?

Hmmmm a nullable Int looks good to me 👍. I'd suggest two things:
- Rename `overlayColor` to `backgroundDimColor`.
- Remove the lambda type. A color integer doesn't need to be lazily evaluated. It was necessary for the `background` Drawable to ensure a new instance is created on each lambda call because Drawables contain shared mutable state.